### PR TITLE
basic components structure

### PR DESCRIPTION
**Pull request by JakPaw1991**

### CODE REVIEW

File: fryzjerappfrontend/src/app/components/login/login.component.ts [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+  public  login: string = "";
+  public password: string = "";
+  public validationMessage: string = "";
+
+  constructor() { }
+
+  public onLogin(): void {
+    this.validationMessage = "";
+    if (this.login.length > 1 && this.password.length > 1) {
+      //wysylanie do serwera
```
[the remark this answers is not in the document]
Comment (JakPaw1991) [reply fryzjerappfrontend/src/app/components/login/login.component.ts]: it's a good way to start:)

File: fryzjerappfrontend/src/assets/business-template/index.html [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+                     <li><a>Kalendarz</a></li>
+                     <li><a>Cennik</a></li>
+                     <li><a>Kontakt</a></li>
+                  </ul>
+               </div>
+
+            </nav>
+         </div>
+      </header>
+      <section>
```
[the remark this answers is not in the document]
Comment (JakPaw1991) [reply fryzjerappfrontend/src/assets/business-template/index.html]: outdated

File: fryzjerappfrontend/src/assets/business-template/index.html [hunk 3]
```diff
@@ -0,0 +1,115 @@
...
+                     <p>Lorem ipsum dolor sit amet, consectetuer adipiscing elit, sed diam nonummy nibh euismod tincidunt ut laoreet dolore magna aliquam erat volutpat.</p>
+                  </div>
+               </div>
+               <div class="s-12 m-6 l-4 margin-bottom">
+                  <div class="box">
+                     <h2>Company</h2>
+                     <p>Lorem ipsum dolor sit amet, consectetuer adipiscing elit, sed diam nonummy nibh euismod tincidunt ut laoreet dolore magna aliquam erat volutpat.</p>
+                  </div>
+               </div>
+               <div class="s-12 m-6 l-4 margin-bottom">
```
[the remark this answers is not in the document]
Comment (JakPaw1991) [reply fryzjerappfrontend/src/assets/business-template/index.html]: outdated

File: fryzjerappfrontend/src/assets/business-template/index.html [hunk 4]
```diff
@@ -0,0 +1,115 @@
...
+               <div class="s-12 l-2">
+                  <img class="s-5 l-12 center" src="img/logo.svg">
+               </div>
+            </div>
+         </div>
+         <!-- TOP NAV -->  
+         <div class="line">
+            <nav class="margin-bottom">
+               <p class="nav-text">Custom menu text</p>
+               <div class="top-nav s-12 l-10">
```
[the remark this answers is not in the document]
Comment (JakPaw1991) [reply fryzjerappfrontend/src/assets/business-template/index.html]: outdated

File: fryzjerappfrontend/src/app/components/carousel/carousel.component.html [hunk 5]
```diff
@@ -0,0 +1,11 @@
+<ngb-carousel *ngIf="images">
+  <div *ngFor="let image of images">
+    <ng-template  ngbSlide>
+      <img class="carousel-image" [src]="image" alt="Random first slide">
+      <!--<div class="carousel-caption">-->
+        <!--<h3>First slide label</h3>-->
```
[the remark this answers is not in the document]
Comment (JakPaw1991) [reply fryzjerappfrontend/src/app/components/carousel/carousel.component.html]: I left it for the future:)(for me)

File: fryzjerappfrontend/src/assets/business-template/index.html (hunk 4 above)
[the remark this answers is not in the document]
Comment (Yunad) [reply fryzjerappfrontend/src/assets/business-template/index.html]: What does outdated mean in this case?

**Pull request merged**